### ISSUE TITLE
File system is too small

### ISSUE THREAD

**Issue by giuliomoro** [LBDonovan/image-builder#14]
`/dev/mmcblk0p1` is  1.6G. Let's aim for 3825204736 bytes (3.82GB) (the size of the eMMC on the black)

**Comment by giuliomoro** [comment]
While we are at it, we should also find a way to make a ~ 100MB FAT16 boot partition

**Comment by LBDonovan** [comment]
Agreed - I'm gonna keep it small while we're developing though just to save on disk space and flashing time :)

**Comment by giuliomoro** [comment]
After installing the things I had to and the man pages I am 1.4GB out of 1.6GB ...
